src/main/java/com/dao/RegistrationDao.java

```diff
@@ -0,0 +1,42 @@
+package com.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import com.model.RegistrationModel;
+import com.util.DBConnection;
+
+/**
+ * Get connection. Insertdata in database. ExecuteUpdate and return value.
+ * 
+ * @author Krushali Talaviya
+ *
+ */
+public class RegistrationDao {
```

Comment:
Please merge the code of RegistrationUserListDao.java to this class so that user-related database operation should be in a single class.